src/components/Reports/DonationsReport/MonthlyActivity/MonthlyActivitySection.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import DonationHistories from '../../../Dashboard/DonationHistories';
+import { GetDashboardQuery } from '../../../../../pages/accountLists/GetDashboard.generated';
+
+interface Props {
+  data: GetDashboardQuery;
+}
+
```

Comment:
Could you add a comment here so that other developers know why some of this is commented out?

```
\\TODO: Query for actual data and load into DonationHistories
```